test-cases/longevity/longevity-mini-test-1h.yaml

```diff
@@ -13,4 +13,4 @@ nemesis_interval: 1
 user_prefix: 'longevity-mini-test'
 space_node_threshold: 6442
 
-append_scylla_args: '--memory 4G'
+#append_scylla_args: '--memory 4G'
```

Comment:
is this related to this PR ? (or intended ?)

Reply:
required for testing - as 4G memory is to little for 8 cores for Scylla (cannot create smaller machines).
I'd gladly increased this value to make this test work for Azure without this change.
For now I'll revert it back as it was just temporary change.